extensions/amp-story-shopping/0.1/amp-story-shopping-config.js

```diff
@@ -8,23 +16,83 @@ export class AmpStoryShoppingConfig extends AMP.BaseElement {
     super(element);
 
     /** @private {string} */
-    this.myText_ = TAG;
+    this.element_ = element;
```

Comment:
We don't need to assign this to a private variable. 
We can remove this and access it by `this.element`.